src/components/Post/Comment/CommentsContainerCard.tsx

```
@@ -0,0 +1,566 @@
+	const { comments, setComments, setTimelines, timelines, overallSentiments, setOverallSentiments } = useCommentDataContext();
+	const isGrantClosed: boolean = Boolean(postType === ProposalType.GRANTS && created_at && dayjs(created_at).isBefore(dayjs().subtract(6, 'days')));
+	const [openLoginModal, setOpenLoginModal] = useState<boolean>(false);
+	const [showOverallSentiment, setShowOverallSentiment] = useState<boolean>(true);
+	const [sentimentsPercentage, setSentimentsPercentage] = useState<ISentimentsPercentage>({ against: 0, for: 0, neutral: 0, slightlyAgainst: 0, slightlyFor: 0 });
+	const [loading, setLoading] = useState(true);
+	const { network } = useNetworkSelector();
+	const [filterSentiments, setFilterSentiments] = useState<ESentiments | null>(null);
+	const router = useRouter();
+	let allComments = Object.values(comments)?.flat() || [];
+	const { resolvedTheme: theme } = useTheme();
+	const [reasonForNoComment, setReasonForNoComment] = useState<String | null>(null);
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)